docs/data/data-grid/recipes-editing/recipes-editing.md

```diff
@@ -113,6 +113,9 @@ You can utilize this callback to batch edits locally and then choose to either p
 
 The demo below stores edited and deleted rows in the `unsavedChangesRef`.
 These changes are saved or discarded when the user clicks the **Save** or **Discard** buttons respectively.
-Row updates from [Clipboard paste](/x/react-data-grid/clipboard/#clipboard-paste) are also batched, as [Clipboard paste uses Editing API for persistence](/x/react-data-grid/clipboard/#persisting-pasted-data).
 
-{{"demo": "BulkEditingNoSnap.js", "bg": "inline", "defaultCodeOpen": false}}
+{{"demo": "BulkEditing.js", "bg": "inline", "defaultCodeOpen": false}}
+
+When using [`DataGridPremium`](/x/react-data-grid/#premium-plan), bulk editing applies to row updates from [Clipboard paste](/x/react-data-grid/clipboard/#clipboard-paste) automatically, since [Clipboard paste uses Editing API for persistence](/x/react-data-grid/clipboard/#persisting-pasted-data):
```

Comment:
maybe convert this into an info callout box?

Reply:
This paragraph is related to the demo right after it, so I wouldn't rather keep it as a regular text.
Or do you mean converting both the paragraph **and** the demo into a callout?